build/ios.js

```diff
@@ -43,35 +43,9 @@ IOS.prototype.clean = function (next) {
 	next();
 };
 
-IOS.prototype.fetchLibTiCore = function (next) {
-	const url = 'http://timobile.appcelerator.com.s3.amazonaws.com/libTiCore-' + TI_CORE_VERSION + '.a.gz',
-		dest = path.join(IOS_LIB, 'libTiCore.a'),
-		markerFile = path.join(IOS_LIB, TI_CORE_VERSION.toString() + '.txt');
-
-	// Do we have the latest libTiCore?
-	if (fs.existsSync(dest) && fs.existsSync(markerFile)) {
-		return next();
-	}
-
-	console.log('You don\'t seem to have the appropriate thirdparty files. I\'ll fetch them.');
-	console.log('This could take awhile.. Might want to grab a cup of Joe or make fun of Nolan.');
-
-	downloadURL(url, TI_CORE_INTEGRITY, function (err, file) {
-		if (err) {
-			return next(err);
-		}
-		gunzip(file, dest, function (err) {
-			if (err) {
-				return next(err);
-			}
-			// Place "marker" file
-			fs.writeFile(markerFile, 'DO NOT DELETE THIS FILE', next);
-		});
-	});
-};
-
 IOS.prototype.build = function (next) {
-	this.fetchLibTiCore(next);
+	// no-op (used to fetch TiCore in the past)
```

Comment:
Clearly not related to this PR, but this jogs my memory. Weren't we going to pre-build libraries for iOS like we do for the other platforms at some point? Do we have JIRA tickets/PRs to track that?

Reply:
I remember you had an angry TODO comment somewhere that states that we should just place libraries inside the repo instead of fetching them manually. In case you mean that, it should be obsolete now that we remove TiCore. Other libraries like libAPSHTTPClient and libAPSAnalytics are prepackaged, just like Hyperloop should / will be in the future.